### DISCUSSION TITLE
[v3] relationManager hasMany

### DISCUSSION THREAD

**Discussion by haringsrob** [filamentphp/filament#7947] [Help]
Hi!

Cannot make it work, so figured I'd ask here.

I have 2 models, user and post.

User hasMany posts ('user_id' (nullable) on the 'posts' table).
Posts belongsTo user (via use_id' on the 'posts' table).

I have a resource 'users' with a relation manager for 'posts' with a relation manager towards posts.

Couple of issues here:
1. when not specifying `inverseRelationship` it tries to get the 'users' relation rather than 'user'. Fixed by setting inverseRelationship('user')
2. when that part is 'fixed', it runs into `Call to undefined method Illuminate\Database\Eloquent\Relations\HasMany::getQualifiedRelatedKeyName()`

Now, I feel this did work before and there may be something super obvious missing on my end.

**Comment by gilr** [comment]
I noticed the same thing (part 1), I needed to define an inverse relationship 'award' to avoid the call of "awards". 
However I didn't see the "part 2" issue, but maybe because I haven't tested much yet.

**Comment by horneberh** [comment]
Hey @haringsrob  I just stumbled over this too... the problem was, that I used an AttachAction on a HasMany relationship, but for this kind of relation you have to use the AssociateAction.
